packages/itwin/grouping-mapping-widget/src/widget/components/provider/ManualUIProvider.scss

```diff
@@ -0,0 +1,17 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+@import "~@itwin/itwinui-css/scss/variables";
+
+.gmw-manual-form {
```

Comment:
Both ManualUIProvider and SearchUIProvider forms share identical styling. Could it reuse the same stylesheet/class or is it a bad practice?

Reply:
I think it's better to separate as it's also an example for user.